app/controllers/admin/stats/top_balances_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Admin::Stats::TopBalancesController < AdminController
+  def index
+    @result = case params[:type]
```

Comment:
Since there's not a default when you click the link on the navbar it results in an exception